<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraceFilterWidget</class>
 <widget class="QFrame" name="TraceFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="tools_panel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="tool_panel" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dark_theme" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="add_model_button">
          <property name="toolTip">
           <string>Add new filter</string>
          </property>
          <property name="text">
           <string>Add new filter</string>
          </property>
          <property name="icon">
           <iconset resource="app.qrc">
            <normaloff>:/icons/add_page</normaloff>:/icons/add_page</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filter_combo_box">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="dark_theme" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="remove_tool_button">
          <property name="toolTip">
           <string>Remove selected</string>
          </property>
          <property name="text">
           <string>Remove selected</string>
          </property>
          <property name="icon">
           <iconset resource="app.qrc">
            <normaloff>:/icons/minus</normaloff>:/icons/minus</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="remove_model_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Remove filter</string>
          </property>
          <property name="text">
           <string>Remove filter</string>
          </property>
          <property name="icon">
           <iconset resource="app.qrc">
            <normaloff>:/icons/cross</normaloff>:/icons/cross</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="FilterTreeView" name="filter_view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterTreeView</class>
   <extends>QTreeView</extends>
   <header>filter_tree_view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
